Mathlib/GroupTheory/CommutingProbability.lean

```diff
@@ -166,9 +166,11 @@ def reciprocalFactors (n : ℕ) : List ℕ :=
   else
     n % 4 * n :: reciprocalFactors (n / 4 + 1)
 
-@[simp] lemma reciprocalFactors_zero : reciprocalFactors 0 = [0] := rfl
+@[simp] lemma reciprocalFactors_zero : reciprocalFactors 0 = [0] :=  by
+  unfold reciprocalFactors; rfl
```

Comment:
Should these use `unseal` so they remain eligible for `dsimp`?

Reply:
I don't have a good sense yet where we need `dsimp` compatibility. We could err on the side of more `unseal`.

Reply:
I guess let's leave them. I doubt this file is used outside of Mathlib yet, so if CI is happy so am I.